.gitignore

```diff
@@ -8,3 +8,4 @@ releases/*.tar.gz
 releases/*.tar.gz.sha256sum
 _output/
 .vagrant/
+.idea/
```

Comment:
I generally recommend to have your personal gitgnore files for IDE/editor-specific rules. Adding them is endless.

https://docs.github.com/en/github/getting-started-with-github/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
yeah, makes sense